httpx/_client.py

```diff
@@ -464,7 +464,10 @@ def _redirect_headers(self, request: Request, url: URL, method: str) -> Headers:
             # Strip Authorization headers when responses are redirected away from
             # the origin.
             headers.pop("Authorization", None)
-            headers["Host"] = url.authority
+
+            # Remove the Host header, so that a new one with be auto-populated on
```

Comment:
Typo here :-) "new one s/with/will/g be"

Reply:
😬 Ya got me